spec/jobs/subscription_placement_job_spec.rb

```diff
@@ -54,9 +54,15 @@
       end
 
       it "processes placeable proxy_orders" do
+        expect(JobLogger.logger).to receive(:info)
         job.perform
         expect(job).to have_received(:place_order).with(proxy_order.reload.order)
       end
+
+      it "logs using JobLogger" do
```

Comment:
I dont see the point of testing the logger calls. but I am fine with it if you think it's worth the extra lines of code :+1:

Reply:
well, me neither. It's something I don't usually test but I needed a way to quickly develop and not boot a worker every time. I can remove them if you prefer. It already served its purposes.

Reply:
ah, I see :+1: it's fine. we keep them.

Reply:
solved